index.html

```diff
@@ -46,21 +46,37 @@ <h2 class="hero-h2"> Who Travels?</h2>
             </div>
         </section>
     </header>
-    <form>
-        <label for="form-destination">Destination:</label><br>
-        <select class="form-destination" name="home-destination">
-            <option value="destination1">Destination #1</option>
-            <option value="destination2">Destination #2</option>
-            <option value="destination3">Destination #3</option>
-        </select><br>
-        <label for="form-time">Time Period:</label><br> <!-- Should this be a dropdown?-->
-        <input type="text" id="form-time" name="time-period"><br>
-        <label for="form-time">Travel Dates:</label><br>
-        <input type="date" id="form-dates" name="travel-dates"><br>
-    </form>
-    <button class="begin-journey">Begin My Journey
-    </button>
-
+    <div class="entire-form-content">
+        <div class="hero-body form-title">
+            <p>Explore a place in time or space</p>
+        </div>
+        <div class="form-block">
+            <form>
+                <div class="search-form">
+                    <div class="search-form-item">
+                        <label for="form-destination">Destination:</label><br>
```

Comment:
If this were an actual site, this would be a text search box rather than a dropdown list. The destinations page would serve as the "let's see where we might go" search. I'm fine either way but this saves filling in something here even though it isn't a functioning search.

Reply:
True -- I was working of the existing code. I'll update.